### PR TITLE
Dwolla Direct integration

### PR DESCRIPTION
**Pull request by matias-aguero-hs**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -155,7 +172,9 @@ def get_key_from_url(url, path_to_remove)
       </label>
     </article>
   <div id="mainContainer">
-     <input type="button" id="start" value="Add Bank">
+     <input type="button" id="start" value="Add Bank by White Label Solution">
+     <br> or <br>
```
Comment (pierre) [inline app.rb:179]: Nice! 💯 

Could you update the screenshots and README too?

Comment (matias-aguero-hs) [reply app.rb:179]: Sure, I will!

**Comment by matias-aguero-hs** [comment]
Hi @pierre 
Any pending task for that PR?

**Comment by pierre** [comment]
Nope, all good!

**Pull request merged**